pages/get-started.tsx

```diff
@@ -125,7 +125,7 @@ export default function GetStartedPage() {
             <Grid container sx={{ alignItems: 'center' }}>
               <ImageContainer
                 alt=""
-                src="/quickstart.svg"
```

Comment:
Can we make sure to delete this picture from the project.

Reply:
Deleted